client/src/components/Configs/index.js

```diff
@@ -1,66 +1,93 @@
-import { useContext, useState } from 'react';
-import { ConfigsContext } from '../../contexts';
+import { useContext, useState } from "react";
+import { ConfigsContext } from "../../contexts";
 
-import "./styles.css"
+import "./styles.css";
 
 const Configs = () => {
-    const { configs, loadTests } = useContext(ConfigsContext);
+  const { configs, loadTests } = useContext(ConfigsContext);
 
-    const [currConfigId, setcurrConfigId] = useState(-1);
-    const [selectedLoadTestId, setselectedLoadTestId] = useState(-1);
-        
-    const configSelect = (id) => () => {
-        currConfigId === id ? setcurrConfigId(-1) : setcurrConfigId(id);
-    }
+  const [currConfigId, setcurrConfigId] = useState(-1);
+  const [selectedLoadTestId, setselectedLoadTestId] = useState(-1);
 
-    const loadTestSelect = (id) => {
-        selectedLoadTestId === id ? setselectedLoadTestId(-1) : setselectedLoadTestId(id);
-    }
+  const configSelect = (id) => () => {
+    return currConfigId === id ? setcurrConfigId(-1) : setcurrConfigId(id);
```

Comment:
Add loadTestSelect to configSelect to handle reset of currConfigId to include automatic reset of currLoadTestId